### DISCUSSION TITLE
personal-development-esfand-1402/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#36] [Announcements]
# personal-development-esfand-1402/

داریم کم‌کم به آخر سال نزدیک می‌شیم و موعدش رسیده که گزارش توسعه فردی این ماه رو منتشر کنم.
صادقانه بگم این ماه از نظر توسعه فردی چندان کار خاصی نکردم و بیشتر فهمیدم کجاها ضعف جدی دارم.
متمم همون روزهای اول ماه بود که پروژه مدل ذهنی متمم رو انجام دادم. اتفاقاً راجع به مدل ذهنی محمدرضا شعبانعلی نوشتم.
کمی هم راجع به تصمیم‌گیری مطالعه کردم.
مصاحبه‌های کاری بالاخره صبح پنج‌شنبه (تقریباً میشه گفت بدترین وقت برای پست گذاشتن در لینکداین)، پست Open to work گذاشتم.

https://aprd.ir/personal-development-esfand-1402/

**Comment by iamamohsen** [comment]
سلام امیر جان، سال جدید رو بهت تبریک میگم.
امیدوارم که امسال، سال خوب، و در انتهای سال داستان های جدید و زیادی برای تعریف کردن داشته باشه. 
از اینکه میتونم گزارش های ماهانه ات رو بخونم خوشحالم، و چه خوب که همزمان داخل تلگرام لینک پست های جدید گذاشته میشه و من میتونم سریع متوجه بشم. امیدوارم که فرصت کنی و همیشه به همین شکل ادامه بدی.

خوندن این نوشته و نوشته های قبلی تو، دو تا سوال درباره متمم خوانی و یک سوال درباره روال زندگی روزانه خودت برای من به وجود اورده بود که خواستم نظرت رو درباره شون بدونم.

یکی درباره سبک تو برای مطالعه متمم هست. من خودم برای اینکار در حال حاضر روشی رو اجرا میکنم و چند باری هم روش ام رو تغییر دادم. اول روی کاغذ مینوشتم و بعد از گذشت چند ماه احساس کردم که در اجرای این روش کمی کمال طلبی اذیتم میکنه، احتمال خودت بهتر بدونی که این موضوع چه معضلی عه. و بعد شروع کردم به نکته برداری با تبلت که این هم خوب بود ولی بعد از چند ماه چون درس ها رو هم با تبلت معمولا میخونم وقتی به دست میگرفتمش ترجیح میدادم به همون درس ها برسم. الان به پیشنهاد یکی از متممی ها، در وان نوت بعد از خوندن هر درس چکیده ای از چیزایی که یاد گرفتم رو اونجا مینویسم. سرعتمم برای مطالعه مقداری بیشتر شده داخل این روش(البته که سعی میکنم اون اصل کند کردن مطالعه رو هم رعایت کنم) . با این حال هنوز خوب بلد نیستم از وان نوت استفاده کنم و دستم سریع نیست. نظر و روش تو چطوره؟ آیا از سبک فعلی متمم خوانی ات راضی هستی؟

و یک موضوع دیگه که اون رو هم دوست داشتم اینجا بنویسم درباره کتاب خوندن هست. من سالی که گذشت بیشتر وقت مطالعه ی آزادم رو به متمم اختصاص دادم تا کتاب های کاغذی، اما احتمالا برای این سال دوباره برگردم به خوندن کتاب ها، احساس میکنم که این کار نعمتی هست که نباید اون رو از دست بدم و جنس مطالعه کتاب یک پکیج کامل تره. در رابطه با این نظرت چیه؟ آیا تو هم متمم رو به کتاب های کاغذی ترجیح میدی یا برعکس؟

و سوال بعدیم درباره تهران نشین بودنت هست. میدونم که اهل یزد هستی. من تا حالا یزد نرفتم. اما میدونم که چقدر استان زیبا و شهر های جالبی داره. همیشه دوست داشتم، برای یک بار هم که شده بیابان گردی کنم. فکر میکنم شهر و استان شما مکان خوبی برای به وقوع پیوستن این خواسته باشه!، و اما بعد ... از نوشته هایت فهمیدم که مدتی رو داخل تهران مشغول زندگی بودی (هستی). من همیشه به اینکه روزی باید برای ادامه تحصیل به تهران سفر کنم، فکر میکنم. اخیراً هم خیلی این فکر کردن زیاد شده. خیلی دوست داشتم که اگر روزی فرصت داشتی، در اینجا درباره تهران بنویسی. از تفاوت هاش، فواید و معایب و از سختی هایی که احتمالا یک جوون مثل من، وقتی بخواد به تنهایی چند سالی اونجا زندگی کنه احتمالا تجربه میکنه. و در کل هر آنچه بنظرت خوبه که این شخص بدونه (که البته همسری هم در کنارش نداره)

پیام طولانی ای شد! مطمئناً انتظار ندارم سریع پاسخ بدی. تا بعد. سال نو مبارک.

**Reply by pourmand1376** [reply]
سلام محسن جان. سوالت طولانی هست و وقت تنگ. 

سعی می‌کنم خلاصه بگم و بعداً سر فرصت بیشتر توضیح بدم. 

اول این که من از روش فعلی متمم‌خوانی‌ام چندان راضی نیستم. قبلاً نوت‌برداری می‌کردم و خیلی متمرکزتر می‌خوندم. اخیرأ خیلی سطحی‌تر شدم ولی حتماً سعی می‌کنم تمرین‌های متمم رو حل کنم؛ یعنی اول یک درس متمم رو می‌خونم و میذارم مطلب تو ذهنم خیس بخوره. بعد از مدتی نامعلوم دوباره به سمتش میام و این دفعه مطلب برام آشناست و سعی می‌کنم تمرینش رو حل کنم. 

دوم این که من کتاب و متمم رو دو نوع مختلف یادگیری می‌دونم و در کنار هم هستند که مفیدن. خود متمم هم دائم به کتاب‌ها برای مطالعه بیشتر ارجاع میده. 

من تو حوزه خودم یه جاهایی مجبورم که ویدئو یوتیوب ببینم. یه سری چیزا رو فقط تو وبلاگ میشه پیدا کرد و یه سری مطالب عمیق‌تر رو فقط تو کتاب‌ها. نهایتاً تو هر حوزه‌ای ناچاریم از منابع متعدد اطلاعات دریافت کنیم و چندان ترجیح ما مهم نیست. چون این‌ها هم پوشانی ندارند. 

ولی اگر ترجیح من رو می‌خوای بدونی، ترجیح میدم که هر محتوایی که مصرف می‌کنم به زبان ساده باشه. از هر محتوایی که بیش ازحد چیزها رو پیچیده می‌کنه، خوشم نمیاد. 

سوم هم راجع به تفاوت‌های تهران و بقیه شهرستان‌ها هست. 

تهران رو از همه شهرستان‌ها جدا می‌کنم چون واقعاً تافته جدا بافته است. بخش بزرگی از صنعت ایران تو تهران شکل گرفته. مثلاً شاید اشتباه نباشه اگر بگیم ۸۰ ۹۰ درصد صنعت نرم‌افزار تو تهران شکل گرفته. قاعدتاً هم جذب نیروی کار بیشتر هست و هم شرایط کاری بهتره تا حدی که حقوق تهران حداقل ۲ ۳ برابر شهرستانه. 

از نظر کاری تهران عالیه. تو هر حوزه‌ای که کار کنی، احتمالاً بهترین‌ها تو تهران هستند و کلی فرصت یادگیری و پیشرفت هست.

اما از بقیه نظرها چندان تعریفی نداره. آدم‌ها همش در حال دویدن هستند. من هر بار سوار مترو می‌شم اعصابم خرد میشه وقتی مردم رو می‌بینم. اکثراً یه حالت افسرده و ناراحتی دارند و در حال بدو بدو برای زندگی هستند. انگار همیشه تو زندگی‌شون دارند، می‌دوند و باز هم به هیچ‌جا نمی‌رسند. 

اگر هم مترو سوار نشی باز ترافیک یه داستان مجزاست و خیلی عادیه که چند ساعت تو روز برای رفت و برگشت تلف بشه. 

البته تهران تفریحات زیادی داره اما اکثراً برای من لذت‌بخش نیستند. نه از فلان وسیله تفریحی تو فلان پارک لذت می‌برم، نه با فلان غذای فلان رستوران عجیب غریب ارتباط برقرار می‌کنم. 

من آرامش خانواده رو دوست دارم و آرامش شهر. 

اگر به ادامه تحصیل فکر می‌کنی اتفاقاً خیلی خوبه که چند سالی بری تجربه کنی. فوقش اینه که خوشت نمیاد و بعد چند سال برمی‌گردی. همین استدلال رو برای خارج کشور هم میشه مطرح کرد. خصوصاً کسی که مجرده و نباید هزینه عجیب غریبی برای جابه‌جایی بده. 

مثلاً چیزی که تو تهران من رو اذیت می‌کرد این‌ بود که ۸ صبح از خونه می‌رفتم بیرون و ۸ شب خسته و مونده می‌رسیدم خونه. بعدشم دیگه حوصله نداشتم کاری بکنم و عملاً زندگی متاهلی تعطیل بود. ولی بچه‌های مجرد این مشکل رو نداشتند. ۳ ۴ ساعت بیشتر تو شرکت می‌موندند و همون‌جا تفریح می‌کردند. مثلاً پلی‌استیشن بازی می‌کردند. 

در تفاوت بین یزد و تهران کلی حرف دیگه هم دارم که الان فرصتش نیست. اگر فرصت بشه به زودی راجع بهش می‌نویسم. 

در ضمن سال نوت رو هم تبریک میگم. امیدوارم سال خوبی داشته باشی.

**Comment by szareui** [comment]
معمولا عادت ندارم اینجا برات بنویسم و هر حرفی داشته باشم رو حضوری بهت میگم، اما اینبار دوست داشتم اینجا بنویسم تا برات بمونه.
با وجود اینکه اکثر چیزایی که می‌نویسی رو شاهدش هستم یا باهات تجربه‌اش کردم اما خوندن پست‌هات برام خیلی با ارزشه، یجورایی هم مرور ماه گذاشته است هم کمک میکنه دنیا رو از نگاه تو ببینم. 
از قشنگی‌های زندگی باهات همین رو بگم که سعی می‌کنی نکات مثبتی که داری رو تقویت کنی و از نکات منفی درس بگیری و راحت از کنارشون رد نمیشی، وقتی مشکلی پیش میاد یا هر کدوم‌مون با چالشی رو برو می‌شیم همکاری میکنی تا باهم حلش کنیم.
برای سال جدید امیدوارم بتونی برای کارهایی که دوست داشتی انجام بدی و بخاطر شرایط نتونستی، وقت بذاری و نتیجش رو توی پست‌هات بخونیم. به خصوص پادکستت که مشتاقانه منتظر اپیزود جدیدش هستم.

**Reply by pourmand1376** [reply]
سلام صدیقه. 

وقتی اینجا کامنت می‌گذاری دیگه نباید انتظار داشته باشی که حضوری بهت جواب بدم :) 

داشتم فکر می‌کردم که چه زود گذشت! ۶ سال از زندگی مشترک‌مون می‌گذره و تو این سال‌ها کلی پیشرفت کردیم و با هم چیزهای مختلف رو تجربه کردیم. 

تو بهم یاد دادی که استریوتایپ‌هایی که راجع به زن‌ها میگن اشتباه هستن (البته تو متمم خونده بودم ولی شنیدن کی بود مانند دیدن). مثلاً میگن خانم‌ها ترسو هستن. در حالی که خیلی جاها بسیار از من شجاع‌تر و جسورتر هستی مخصوصاً تو گفتگوهای سخت و تصمیم‌گیری‌های سخت که از خدایان این حوزه هستی. 

راستی، کاش میشد تو هم یه اکانت ویرگول درست کنی و توش گزارش ماهانه منتشر کنی که منم همین‌طور بیام بخونم و لذت ببرم.

**Comment by mrshabanali** [comment]
امیر جان.
فقط خواستم این‌جا کامنت گذاشته باشم که کامنت گذاشته باشم.
و بهانه‌ام هم این‌که پروژهٔ مدل ذهنی رو خوندم و لذت بردم. لذت از این جهت که دیدم چقدر خوب و دقیق توضیح داده بودی و توصیف کرده بودی.
و باعث شد موضوعی که چند ساله با شواهد متعدد در ذهن من جا افتاده، بیش از پیش تقویت بشه. این‌که شناختن آدم‌ها از طریق نوشته‌ها و حرف‌هاشون - به فرض که به اندازهٔ کافی صداقت و خودافشایی داشته باشن - می‌تونه حتی خیلی موثرتر از شناختن حاصل از تعامل رودررو باشه.
بین دوستانی که باهاشون تعامل فیزیکی و دیدار و گفتگو دارم، و احتمالاً خودشون رو جزو دوستان نزدیکم می‌دونن، آدم‌های متعددی رو می‌شناسم که اصلاً به اندازهٔ تو به خودم نزدیک نمی‌بینم‌شون. و حس می‌کنم در خوش‌بینانه‌ترین حالت، «آشنای دور» اون‌ها هستم. در حالی که تو در نگاه من دوست نزدیکی (خودم هم با خوندن منظم نوشته‌ها و گزارش‌هات، همین حس نزدیکی رو دارم).

**Reply by pourmand1376** [reply]
سلام محمدرضا. 

از دیدن پیامت خیلی خوشحال شدم و ذوق کردم. اتفاقاً همراه خانمم کنار ساحل نشسته بودیم. گفت داری چیکار می‌کنی. گفتم «فکر کنم شعبانعلی کامنت گذاشته برام». می‌خواست گوشی رو از دستم بکشه که گفتم «بابا من خودمم نخوندم، بیا با هم بخونیم :)». 

خیلی برام ارزشمنده که میگی حرف‌هام رو می‌خونی. خصوصاً این چند وقت تو تک‌تک تصمیم‌گیری‌هام به حرف‌هات فکر می‌کردم و سعی می‌کردم نکاتی که میگی رو رعایت کنم. انصافاً هم خیلی خوب شد و شرایط کاریم الان خیلی بهتر از قبله‌. 

برام افتخاره که من رو دوست خودت می‌دونی. راستی اگر زمانی یزد اومدی حتماً یه خبر بهم بده. بالاخره یه نون و پنیری هست که دور هم بخوریم و کویر یزد هم خیلی قشنگه. من خودم آرامش کویر رو خیلی دوست دارم و حدس می‌زنم تو هم دوست داشته باشی. 

اینم بگم که اگر خواستی یزد بیای، ترجیحاً رمضون نباشه. شهر مذهبی هست و متاسفانه برای رمضون سخت می‌گیرند و خودمون هم اذیت میشیم، چه برسه به مسافر (تو تهران این مشکل خیلی خیلی کمتره).

خلاصه کلام از پیامت خیلی کیفور شدم. بازم از این کارا بکن.
